src/consensus/constants.py

```diff
@@ -62,6 +62,7 @@ class ConsensusConstants:
     BLOCKS_CACHE_SIZE: uint32
     NETWORK_TYPE: int
     MAX_GENERATOR_SIZE: uint32
+    MAX_GENERATOR_REF_LIST_SIZE: uint32
```

Comment:
Consider changing to `MAX_GENERATOR_REF_LIST_COUNT` so mypy can help you be sure you got all the semantic changes.